### ISSUE TITLE
Crashing again...this time won't let me back in. 

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Metamorphoses#17]
I noticed I had been getting a lot of null point exceptions, but I could restart the world and it would be fine...any idea's on what caused this crash that won't let me back in? 
[crash-2019-07-03_10.56.34-server.txt](https://github.com/teagan75/Metamorphoses/files/3355551/crash-2019-07-03_10.56.34-server.txt)

**Comment by teagan75** [comment]
I has something to do with lore expansion. Did you just get a new lore
note? Did they all disappear again?

**Comment by Ladysarajane** [comment]
Yeah....I just completed the Hamlet quest and the reward was a Lore page. As soon as I clicked claim the game crashed

It has been a couple days..since I looked at the lore book and the pages were there then. but I had died since then and I never looked

**Comment by teagan75** [comment]
To solve this you have two options: 1. you can delete your player data, get in and run the command /lore reload (this will clear your inventory and maybe other data and should be a last resort)  or (recommended) 2. restore a backup, get in and then try running the command /lore reload, then complete the Hamlet chapter

I changed and added new lore and I wonder if that messed up worlds already made

**Comment by Ladysarajane** [comment]
Yeah.....dumb me hasn't backed up in two days....ugh, but that would go with the new update and getting the new page and having it crash. It might be better to go to that backup and running the command /lore reload. I will let you know how it goes later this afternoon.

**Comment by Ladysarajane** [comment]
My ideal worked! LOL...I used twitch to "turn off lore expansions" and when I loaded the game and went into the world, the Lore journal was gone..but the world loaded. Shut down game...turned the mod back on in twitch and reloaded the game and was still able to get in...I used the command /lore reload and went into creative and and gave myself the journal and pages and I seem to be fine playing again. 

Thanks for the help! I couldn't figure out what caused the crash. 

Edit: does this give you any info or is it because it is already claimed?
![2019-07-03_11 36 17](https://user-images.githubusercontent.com/34258851/60605038-c94dac00-9d86-11e9-854c-a3084926cef1.png)

**Comment by teagan75** [comment]
Glad it worked!!

**Comment by teagan75** [comment]
Oh and the world should be saving automatically with FTB Utils. All your backups should be in the backup folder in your instance

**Comment by Ladysarajane** [comment]
For some reason there wasn't a back up in that folder for a couple days. Not sure why FTB is  not doing the backups regularly.

**Comment by teagan75** [comment]
You can double check the config. It’s found in the local folder. Under “backups” enabled should be true

**Comment by Ladysarajane** [comment]
All true...I think I had been getting off and on before backups made

**Comment by teagan75** [comment]
I hope the ticking entity crashes are solved now in 0.4.2_hotfix

Apparently, the launcher didn't properly copy my files and they were missing lore pages - maybe other things as well - so hopefully no more issues.

**Comment by Ladysarajane** [comment]
Great. I will try it in the am. Saw your tweet about. I'll check the change log.

**Comment by Ladysarajane** [comment]
Okay...I ran the update, re-ran the commands and all seems to be smooth...No ticking entities, Lore pages all intact. I think this is solved for me. Thanks for the quick fix! Oh my backup does work. I played it long enough, I saw it and confirmed that it showed in my backup folder. My suspicion was that I hadn't played enough in my previous sessions to trigger it to run. Happy 4th! Hope JJ has fun with watching the fireworks! I won't be watching them....I have two fearful doggies that need TLC during them. Thunderstorms don't bother them....I don't know why they don't like the firework booms.
